server/routers/project.feature.router.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const getDB = require('../database/bootstrap.database');
+
+const projectFeatureRouter = express.Router();
+
+projectFeatureRouter.post('/:projectid/create/feature', (req, res) => {
+    const projectid = req.params.projectid
+    const { feature_data } = req.body;
+    const db = getDB();
+    db.create_project_feature([ projectid, feature_data] )
+        .then( promise => res.send())
+        .catch(err => res.send(err));
+    
+});
+
+projectFeatureRouter.get('/:projectid/feature/:featureid', (req, res) => {
+    const projectid = req.params.projectid;
+    const featureid = req.params.featureid;
+    const db = getDB();
+    db.find_project_feature([ featureid ])
+        .then( promise => res.send() )
```

Comment:
since this is a GET request, we are expecting to get something specific from the database, so I would label the promise what we're expecting, in this case `projectFeature` and then have the RES send that to the front end.

my code:
`.then( projectFeature => res.send(projectFeature) )`

Let me know if you have questions.